cert-manager/cert-manager.yaml

```diff
@@ -433,183 +1491,6 @@ spec:
               memory: 32Mi
             
 
----
-# Source: cert-manager/charts/webhook/templates/ca-sync.yaml
-## This file contains a CronJob that runs every week to automatically update the
-## caBundle set on the APIService and ValidatingWebhookConfiguration resource.
-## This allows us to store the CA bundle in a Secret resource which is
-## generated by cert-manager's 'selfsigned' Issuer.
-apiVersion: batch/v1beta1
-kind: CronJob
-metadata:
-  name: cert-manager-webhook-ca-sync
```

Comment:
how do we deal with resources like this that were deleted?

Reply:
Not sure on the best way to handle this here - I assume this is deployed with some equivalent of `kubectl apply`?

I think in this instance, these resources will need to be manually deleted 🙈

Reply:
yeah, we're just using `kubectl apply` here...